tiatoolbox/visualization/tileserver.py

```diff
@@ -47,7 +51,39 @@ def __init__(self, title: str, layers: Dict[str, WSIReader]) -> None:
             static_folder=data._local_sample_path(Path("visualization") / "static"),
         )
         self.tia_title = title
-        self.tia_layers = layers
+
+        # generic layer names if none provided
+        if isinstance(layers, list):
+            layers = {f"layer-{i}": p for i, p in enumerate(layers)}
+        # set up the layer dict
+        layer_def = {}
+        meta = None
+        for i, key in enumerate(layers.keys()):
```

Comment:
@measty This could be better written as just `enumerate(layers)` as this will enumerate the keys. This should have been flagged with pre-commit. Are you running pre-commit before committing?